### DISCUSSION TITLE
NFC Stickers?

### DISCUSSION THREAD

**Discussion by beyondi-ultra** [MiczFlor/RPi-Jukebox-RFID#1505] [Q&A]
After many hours of tinkering and wood working I finally managed to get my Phoniebox up and running. 
I thought it would be a good idea to use stickers as opposed to cards for mine because I wanted to only tag physical CD's and that simply seemed the easiest.

I have posted this on the gitter chat as well, but maybe its more likely here to get a response?
It might be a real stupid question but how does the Phoniebox work with NFC stickers? I have a bunch of stickers here and can assign different actions to them if I use my iphone and an NFC Tagwriter app, but if I scan them with the phoniebox they all have the same chip ID. I feel like I am missing something super obvious. Anyone here who can help with that?
So basically when I assign an album to one sticker, all the other stickers will start playing the same album as well when used on the nfc reader.

Am I using [the wrong stickers](https://www.amazon.co.uk/gp/product/B00M5PC0FG/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)? 

Any help would be much appreciated.
